src/App.js

```diff
@@ -1,15 +1,61 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import './App.css';
 
-import Navbar from '../src/components/Navbar/Navbar'
-import Map from '../src/components/Map/Map'
+import Map from '../src/components/Map/Map';
+import Modal from '../src/components/Modal/Modal';
+import Navbar from '../src/components/Navbar/Navbar';
 
 
 function App() {
+  let [appState, setAppState] = useState({ showModal: true, geolocationPermission: false });
```

Comment:
Are we changing these values anywhere? If not you might use const instead

Reply:
Correct, changed to const!